qos/evm/check.go

```diff
@@ -0,0 +1,52 @@
+package evm
+
+import (
+	"github.com/buildwithgrove/path/gateway"
+	"github.com/buildwithgrove/path/qos/jsonrpc"
+	"github.com/buildwithgrove/path/relayer"
+)
+
+var (
+	idChainIDCheck     = jsonrpc.IDFromInt(1001)
```

Comment:
What's 1001? Is this an `iota` sort of situation?

Reply:
Thanks, fixed.